scylla-cql/src/types/deserialize/value.rs

```diff
@@ -3704,3 +3704,16 @@ fn _test_udt_bad_attributes_rename_collision_with_field() {}
 /// }
 /// ```
 fn _test_udt_bad_attributes_rename_collision_with_another_rename() {}
+
+/// ```compile_fail
+///
+/// #[derive(scylla_macros::DeserializeValue)]
+/// #[scylla(crate = crate)]
+/// struct TestUdt {
+///     a: i32,
+///     #[scylla(allow_missing)]
+///     b: bool,
+///     c: String,
+/// }
+/// ```
+fn _test_udt_bad_attributes_name_enforce_order_limitations_on_allow_missing() {}
```

Comment:
A follow-up question: are such test cases executed somewhere? I'm not familiar with the doc tests. I'm curious since CI passed, even though this test is not correct IMO.

Reply:
Apparently, all those doc tests fail because of an unresolved import:
```rust
rror[E0433]: failed to resolve: unresolved import
 --> scylla-cql/src/types/deserialize/value.rs:1957:10
  |
7 | #[derive(DeserializeValue)]
  |          ^^^^^^^^^^^^^^^^ unresolved import
  |
```
trying to understand why.

Reply:
The reason was putting `crate = crate` instead of `crate = scylla_cql`. Doc tests are run from outside the crate's context, that's why it didn't work.

Reply:
> Why should this fail? I don't see enforce_order and skip_name_checks attributes defined (and this should fail only in this case, as I understood from the commit message).

Good catch, I've added those attributes, along with a "control sample" test which is expected to compile successfully.